helpers/expoNotifications.js

```diff
@@ -0,0 +1,87 @@
+const Expo = require('expo-server-sdk');
```

Comment:
We use 2 spaces as the indentation. This file uses tabs. Can you update it?

Reply:
Sure. I'll also add an editorconfig.